Validation/RecoTau/plugins/DQMHistNormalizer.cc

```diff
@@ -18,15 +18,30 @@
 #include "FWCore/ServiceRegistry/interface/Service.h"
 
 //Regexp handling
-#include "classlib/utils/RegexpMatch.h"
-#include "classlib/utils/Regexp.h"
-
+#include <regex>
 #include <string>
 #include <vector>
 #include <map>
 
 using namespace std;
 
+// Three implementations were tested: char-by-char (this version),
+// using std::string::find + std::string::replace and std::regex_replace.
+// First one takes ~60 ns per iteration, second one ~85 ns,
+// and the regex implementation takes nearly 1 us
+std::string globToRegex(const std::string& s) {
```

Comment:
This function would be better in anonymous namespace or `static` to avoid name collisions.